### PR TITLE
update fake to rc_1

### LINKED ISSUE #105: Update to latest FAKE beta
@matthid please help

### PR DESCRIPTION
**Pull request by matthid**
fixes #105

### CODE REVIEW

File: src/Mechanic/Mechanic.fsproj [hunk 1]
```diff
@@ -4,7 +4,7 @@
     <TargetFramework>netstandard2.0</TargetFramework>
   </PropertyGroup>
   <ItemGroup>
-    <Compile Include="../../paket-files/Microsoft/visualfsharp/src/fsharp/FSharp.Build/Fsc.fs">
+    <Compile Include="..\..\paket-files\Microsoft\visualfsharp\src\fsharp\FSharp.Build\Fsc.fs">
```
Comment (tlycken) [inline src/Mechanic/Mechanic.fsproj]: This might break the build on Linux.

Comment (matthid) [reply src/Mechanic/Mechanic.fsproj]: Oh thanks. I reverted that but some automatic tool seems to have done it again :/

Comment (forki) [reply src/Mechanic/Mechanic.fsproj]: \ are fine in MSBuild on linux

**Comment by forki** [comment]
thanks!

**Pull request merged**